python/pyspark/ml/torch/tests/test_distributor.py

```diff
@@ -271,6 +278,7 @@ def test_create_torchrun_command(self) -> None:
         self.delete_env_vars(input_env_vars)
 
 
+@unittest.skipIf(not have_torch, "torch is required")  # type: ignore
```

Comment:
actually we should exclude this in mypy test.I will do it separately.

Reply:
Thank you!